### PR TITLE
Feature/Inline Cross Editing

### PR DESCRIPTION
**Pull request by ogbiyoyosky**
Ref: #287

Completes:

Moderators should be able to edit inline:

a. Repository
b. Categories, even from contributions to task requests.
c. Tags.

### CODE REVIEW

**Comment by azarus** [comment]
LGTM

File: src/components/Story/StoryFull.js [hunk 1]
```diff
@@ -321,7 +325,6 @@ class StoryFull extends React.Component {
     const repository = metaData.repository;
     const postType = post.json_metadata.type;
     const alreadyChecked = isModerator && (post.reviewed || post.pending || post.flagged);
-    const unreviewed = !post.reviewed && !post.pending && !post.flagged;
```
Comment (eastmaels) [inline src/components/Story/StoryFull.js]: I added this for another issue. This most probably caused a merge conflict and codes were overwritten. Please check.

File: src/components/Story/StoryFull.js [hunk 2]
```diff
@@ -383,20 +386,24 @@ class StoryFull extends React.Component {
...
+          {isModerator && alreadyChecked ?
           <div>
-            {!mobileView ? 
+            {!mobileView ?
             <span>
             <h3><center><Icon type="safety" /> Moderation Control </center></h3>
-            {<p><b>Moderated By: &nbsp;</b> <Link className="StoryFull__modlink" to={`/@${post.moderator}`}>@{post.moderator}</Link></p>}
+            {post.reviewed && <p><b>Status: &nbsp;</b> <Icon type="check-circle"/>&nbsp; Accepted <span className="smallBr"><br /></span> <b>Moderated By: &nbsp;</b> <Link className="StoryFull__modlink" to={`/@${post.moderator}`}>@{post.moderator}</Link></p>}
+            {post.flagged && <p><b>Status: &nbsp;</b> <Icon type="exclamation-circle"/>&nbsp; Hidden <span className="smallBr"><br /></span> <b>Moderated By: &nbsp;</b> <Link className="StoryFull__modlink" to={`/@${post.moderator}`}>@{post.moderator}</Link></p>}
+            {post.pending && <p><b>Status: &nbsp;</b> <Icon type="sync"/>&nbsp; Pending <span className="smallBr"><br/></span> <b>Moderated By: &nbsp;</b> <Link className="StoryFull__modlink" to={`/@${post.moderator}`}>@{post.moderator}</Link></p>}
```
Comment (eastmaels) [inline src/components/Story/StoryFull.js]: These last 4 lines of codes as well.

File: src/components/Story/StoryFull.js [hunk 3]
```diff
@@ -383,20 +386,24 @@ class StoryFull extends React.Component {
...
             <span>
             <h3><center><Icon type="safety" /> Moderation  </center></h3>
-            {<p> <b>Mod: &nbsp;</b> <Link className="StoryFull__modlink" to={`/@${post.moderator}`}>@{post.moderator}</Link></p>}
+            {post.reviewed && <p> <Icon type="check-circle"/>&nbsp; Accepted <span className="smallBr"><br /></span> <b>Mod: &nbsp;</b> <Link className="StoryFull__modlink" to={`/@${post.moderator}`}>@{post.moderator}</Link></p>}
+            {post.flagged && <p> <Icon type="exclamation-circle"/>&nbsp; Hidden <span className="smallBr"><br /></span> <b>Mod: &nbsp;</b> <Link className="StoryFull__modlink" to={`/@${post.moderator}`}>@{post.moderator}</Link></p>}
+            {post.pending && <p> <Icon type="sync"/>&nbsp; Pending <span className="smallBr"><br/></span> <b>Mod: &nbsp;</b> <Link className="StoryFull__modlink" to={`/@${post.moderator}`}>@{post.moderator}</Link></p>}
             </span>
             }
-          </div> 
+          </div>
```
Comment (eastmaels) [inline src/components/Story/StoryFull.js]: These 6 lines too were overwritten.

File: src/components/Story/StoryFull.js [hunk 4]
```diff
@@ -443,25 +450,8 @@ class StoryFull extends React.Component {
...
-          :
-          <span>
-            {post.reviewed && <p> <Icon type="check-circle"/>&nbsp; Accepted <span className="smallBr"><br /></span> </p>}
-            {post.flagged && <p> <Icon type="exclamation-circle"/>&nbsp; Hidden <span className="smallBr"><br /></span> </p>}
-            {post.pending && <p> <Icon type="sync"/>&nbsp; Pending <span className="smallBr"><br/></span> </p>}
-            {unreviewed && <p> <Icon type="eye-o"/>&nbsp; Not Reviewed <span className="smallBr"><br/></span> </p>}
-          </span>
-          }
-        </div>
+        </div> : null}
```
Comment (eastmaels) [inline src/components/Story/StoryFull.js]: And these until line 446. I'm okay to resolve these merge miss once this PR is merged to develop. Will just re-add the above lines after the merge.

**Comment by ogbiyoyosky** [comment]
@eastmaels I fixed the conflict.

**Comment by eastmaels** [comment]
Thank you @ogbiyoyoski. Looks good to me.

**Pull request closed without merging**